view-user/src/components/common/PrizeCard/PrizeCard.module.css

```
@@ -0,0 +1,57 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.card {
+  /* TODO　cardのみにオーバレイがかかっているがどうするべきか意見が欲しい。 */
+  position: relative;
+  width: 25vw;
+  height: 25vw;
+  border: 1.25vw solid #ff3342;
+  border-radius: 3.125vw;
+  box-sizing: border-box;
+  box-shadow: 0.9375vw 0.9375vw 0px #ff3342;
+}
+
+.image {
+  /* TODO　画像の丸みとカードの丸みが合わない。。。 */
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+  border-radius: 2vw;
+  margin-bottom: 2.5vw;
+}
+
+.text {
+  color: #ff3342;
+  font-family: "Noto Sans JP";
+  font-size: 2.8125vw;
+  font-weight: 500;
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border-radius: 3.125vw;
```

隙間を考えるともう少し小さいほうが良さそうです。
```suggestion
  border-radius: 1.8vw;
```